### ISSUE TITLE
Accessibility: Improve screenreader support

### ISSUE THREAD

**Issue by RastislavKish** [linkwarden/linkwarden#1368]
Hello! 👋

First of all, thanks for this great product!

I'm a screenreader user, and I wonder if there are any plans on improving the Linkwarden interface accessibility. Reviewing the demo instance, there are quite a few a11y issues that could be resolved with little modifications and tweaks:

* Missing headings: Important sections of the page, but especially the individual bookmarks could be marked as headings, so the users can quickly move around.
* Non-standard HTML components: If possible, use the semantic UI components provided by the HTML standard, they are battle tested and typically provide optimal aria representation, keyboard accessibility and behavior out of the box. Better functionality, less work!
* Unlabelled buttons: Controls which lack a textual description could use the aria-label attribute to describe their purpose, so the screenreader will announce them properly.
* Keyboard accessibility: If possible, make sure the interface with its core functions can be used from the keyboard, i.e. all options and menus are reachable using keyboard navigation.

These are just some of the things that come to my mind when looking at the app. Reading other issues, it seems there's a lot of work on making the interface as convenient to use as possible, and I believe improving the a11y would be a great move in this initiative, enhancing the project reputation, improving code maintainability and of course bringing linkwarden to more users.

**Comment by seals187** [comment]
Hey there, 

I'm not using a screenreader, and what you are saying makes 100% sense, as I'm not 
contributing, I hope they will make these changes for screenreader users.
